### ISSUE TITLE
model.Eval() test trainData got a huge difference against last epoch training

### ISSUE THREAD

**Issue by vivid216** [dotnet/TorchSharp#470]
>   I print the last training epoch output predicted and target. directly use this  model，call `model.Eval()`  and then test the same training data , expect the ouput will be overfitting,   but ouput predicted is huge  difference  vs the target.  

> someone says it caused by batch_normalization or dropout layers like this [performance-highly-degraded-when-eval-is-activated-in-the-test-phase](https://discuss.pytorch.org/t/performance-highly-degraded-when-eval-is-activated-in-the-test-phase/3323/10 ),  but i removed normalization layer,  the problem not solved.  i don't know how to use the training result. 

> this is my model code: and torchsharp version is **0.95.3**

  ```
            modules = nn.ModuleDict();
            // input -1,1024,25
            modules.Add(($"conv1d-first", nn.Conv1d(1024, 512, kernelSize: 3, stride: 1, padding: 0L, bias: true))); //(512,25-3+1 + 0)
                                                                                                                     // modules.Add(($"bnrm1d-first", BatchNorm1d(512,track_running_stats:false))); //(512,23)
            modules.Add(($"relu-first", ELU(inPlace: true)));
            modules.Add(($"conv1d-second", nn.Conv1d(512, 1024, kernelSize: 3, stride: 1, padding: 0L, bias: true))); //(1024,23-3+1 + 0)
            modules.Add(($"relu-second", ELU(inPlace: true)));
            modules.Add(("avgpool", AvgPool1d(2))); //1024,(21-1)/2
            modules.Add(($"lstm-first", new LocalLSTM("lstm", 10, 64, 1, batchFirst: false, bidirectional: true, device: device)));
            modules.Add(("flatten", Flatten()));
            modules.Add(($"linear", Linear(64 * 2, 1)));

            layers = Sequential(modules.values());
            RegisterComponents();

            if (device != null && device.type == DeviceType.CUDA)
                this.to(device);
```
